Source/Shaders/Builtin/Functions/windowToEyeCoordinates.glsl

```diff
@@ -30,6 +30,27 @@ vec4 czm_windowToEyeCoordinates(vec4 fragmentCoordinate)
     float z = (fragmentCoordinate.z - czm_viewportTransformation[3][2]) / czm_viewportTransformation[2][2];
     vec4 q = vec4(x, y, z, 1.0);
     q /= fragmentCoordinate.w;
-    q = czm_inverseProjection * q;
+
+    bool usingOrthographic = all(equal(czm_inverseProjection[0], vec4(0.0))) &&
+                             all(equal(czm_inverseProjection[1], vec4(0.0))) &&
+                             all(equal(czm_inverseProjection[2], vec4(0.0))) &&
+                             all(equal(czm_inverseProjection[3], vec4(0.0)));
```

Comment:
I think this can just be `bool usingOrthographic = czm_inverseProjection == mat4(0.0)`